### PR TITLE
[TPv1] Added a new route and logic for assigning multiple server capabilities to a server

### PR DESCRIPTION
**Pull request by rimashah25**
Closes: https://github.com/apache/trafficcontrol/issues/6033

<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?

1. Once you create a server, click on `Manage Capabilities`
2. To assign multiple capabilities to a server, click on image
3. A new dialog window listing all server capabilities (and the one assigned to the existing server as checked) should open up, like this:
image
4. You can chose from the list and click submit and then see it update on the server/<server_name>/capabilties table.

## PR submission checklist
- [ ] This PR has tests 
- [ ] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: traffic_portal/app/src/scripts/config.js [hunk 1]
```diff
@@ -23,4 +23,4 @@
 
 angular.module('config', [])
 
-.constant('ENV', { api: { unstable:'/api/4.0/', stable: "/api/3.1/" } });
+.constant('ENV', { api: { unstable:'/api/4.1/', stable: "/api/3.1/" } });
```
Comment (srijeet0406) [inline traffic_portal/app/src/scripts/config.js:26]: Shouldn't the stable be `4.0` now?

File: traffic_portal/app/src/common/modules/table/serverServerCapabilities/table.assignServerSCs.tpl.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <h3 class="modal-title">Assign Server Capabilities to {{::server[0].hostName}}</h3>
+</div>
+<div class="modal-body">
+    <common-grid-controller table-name="serverSCsUnassigned" options="gridOptions"
+                            data="serverCapabilities" columns="columns" selected-data="selectedSCs"></common-grid-controller>
+</div>
+<div class="modal-footer">
+    <button class="btn btn-link" ng-click="cancel()">cancel</button>
+    <button class="btn btn-primary" ng-click="submit()">Submit</button>
+</div>
```
Comment (srijeet0406) [inline traffic_portal/app/src/common/modules/table/serverServerCapabilities/table.assignServerSCs.tpl.html:31]: nit: missing blank line at the end of the file

File: traffic_portal/app/src/common/modules/table/serverServerCapabilities/TableAssignServerSCsController.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+    /** @type CGC.GridSettings */
+    $scope.gridOptions = {
+        selectRows: true,
+        selectionProperty: "selected"
+    };
+};
+
+TableAssignServerSCsController.$inject = ['server', 'serverCapabilities', 'assignedSCs', '$scope', '$uibModalInstance'];
+module.exports = TableAssignServerSCsController;
```
Comment (srijeet0406) [inline traffic_portal/app/src/common/modules/table/serverServerCapabilities/TableAssignServerSCsController.js:63]: nit: missing blank line at the end

File: traffic_portal/app/src/scripts/config.js (hunk 1 above)
Comment (rimashah25) [reply traffic_portal/app/src/scripts/config.js:26]: yeah, we can change that to 4.0..

**Review by srijeet0406 — APPROVED** [review]
Code looks good. Tested manually and everything seems to work fine. All tests pass.

**Pull request merged**